easypost.js file is missing in the build

```
0|main  | Error: Cannot find module './easypost.js'
0|main  | Require stack:
0|main  | - /home/ubuntu/node_modules/@easypost/api/index.js
```
Index.js file is importing easypost.js which is not present in the api directory.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,12 @@
 ## CHANGELOG
 
+### 3.11.1 2021-06-11
+
+* Re-package to fix missing file
+* add "prepublishOnly" script and ignore all build assets (#177)
+* fix: switch "package" to "prepublishOnly"
+* docs: adds info about built docs to README
+
 ### 3.11.0 2021-06-10
 
 * Adds `SmartRate` functionality to the `Shipments` object (available by calling `getSmartrates()` on a shipment)

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,7 +4,6 @@
 
 * Re-package to fix missing file
 * add "prepublishOnly" script and ignore all build assets (#177)
-* fix: switch "package" to "prepublishOnly"
 * docs: adds info about built docs to README
 
 ### 3.11.0 2021-06-10

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "@easypost/api",
   "description": "EasyPost Node Client Library",
-  "version": "3.11.0",
+  "version": "3.11.1",
   "author": "Easypost Engineering <support@easypost.com>",
   "homepage": "https://easypost.com",
   "bin": {

